src/courses/tests/test_views.py

```diff
@@ -34,19 +34,19 @@ def test_all_rank_page_status_codes_for_anonymous(self):
         ''' If not logged in then all views should redirect to home page or admin login '''
 
         self.assertRedirectsLogin('courses:ranks')
-        self.assertRedirectsAdmin('courses:rank_create')
-        self.assertRedirectsAdmin('courses:rank_update', args=[1])
-        self.assertRedirectsAdmin('courses:rank_delete', args=[1])
+        self.assertRedirectsLogin('courses:rank_create')
+        self.assertRedirectsLogin('courses:rank_update', args=[1])
+        self.assertRedirectsLogin('courses:rank_delete', args=[1])
 
     def test_all_rank_page_status_codes_for_students(self):
-        ''' If logged in as student then all views except ranks list view should redirect to home page or admin login '''
+        ''' If logged in as student then all views except ranks list view should redirect to home page '''
```

Comment:
nothing in this test redirects to home page

Reply:
fixed